manifest.yml

```diff
@@ -0,0 +1,14 @@
+---
+applications:
+  - name: stub-oidc-client
+    memory: 1G
+    buildpacks:
+      - java_buildpack
+    command: (cd stub-oidc-client && ls -l &&  bin/stub-oidc-client server ./stub-oidc-client.yml)
+    env:
+      JAVA_HOME: "../.java-buildpack/open_jdk_jre"
+      STUB_OP_URI: https://stub-oidc-op.cloudapps.digital
```

Comment:
Not for now, but we could make the op app private and then use stub-oidc-op.apps.private as it's endpoint. Same for the verifiable credential app.

Reply:
Good shout 👍 That would be better!